toolset/utils/docker_helper.py

```diff
@@ -279,6 +279,13 @@ def stop(benchmarker_config=None,
     client.containers.prune()
     client.networks.prune()
     client.volumes.prune()
+    if benchmarker_config.server_docker_host != benchmarker_config.database_docker_host:
+        db_client = docker.DockerClient(
+            base_url=benchmarker_config.database_docker_host)
+        db_client.images.prune()
+        db_client.containers.prune()
+        db_client.networks.prune()
+        db_client.volumes.prune()
```

Comment:
Another thing I'm curious to test is line 277 above this change. Instead of opening `db_client`, I just pass the container originally created from a `db_client` down to here, and then call `database_container.stop()`. I wonder if it is clever enough to use the original client rather than one freshly created. It may be best to just err on the side of caution and move those three lines (including the comment) into this block, and instead call `db_client.containers.stop(database_container.id)`